stream/index.html

```diff
@@ -44,13 +44,30 @@
 
   <div class="app">
     <div class="kchung">
-      <div class="play-pause">
-        <a href="" class="play-button no-hover">
-          <i class="fa fa-play big-icon" aria-hidden="true"></i>
-        </a>
-        <a href="" class="pause-button hidden no-hover">
-          <i class="fa fa-pause big-icon" aria-hidden="true"></i>
-        </a>
+      <div class="play-buttons-container">
```

Comment:
Two play buttons on the same page seems a bit weird for UX. What if we utilized stream two for the voscast stream? Then we could change the link to say something about the Getty.

Reply:
Yeah I agree - though the ux here was requested by Kern. 

But I'm having issues with using the `audio` element with the voscast URL (401 auth error) & that I've troubleshooted quite a bit, seemed like the solution from voscast documentation was to use their 'flashplayer'. That player element will likely look different so maybe using the stream2 page would be good.

Though I'm blocked by the voscast credentials I was given not working to connect to the server

Reply:
Putting the public stream on stream2 would remove most of the complexity from this PR I think which would be great

Reply:
Actually yeah Kern is saying we should have it here still on the main page and that I should just publish it now; I think we can revert this PR after the stream but for now it seems somewhat urgent